packages/firebase_app_check/firebase_app_check/android/src/main/java/io/flutter/plugins/firebase/appcheck/FlutterFirebaseAppCheckPlugin.java

```diff
@@ -80,6 +81,8 @@ private Task<Void> activate(Map<String, Object> arguments) {
             FirebaseAppCheck firebaseAppCheck = getAppCheck(arguments);
             firebaseAppCheck.installAppCheckProviderFactory(
                 SafetyNetAppCheckProviderFactory.getInstance());
```

Comment:
I feel like SafetyNet should be behind a flag. [SafetyNet is eventually being deprecated](https://developer.android.com/training/safetynet/deprecation-timeline) and the Play Console is warning folks about SafetyNet being present in their APKs and AABs. I think PlayIntegrityAPI should be the default choice and then optionally have a flag that can switch over to safetyNet, but not have both appcheck providers active at the same time since both provide attestation.